### PR TITLE
v11 Embryonal modify and rerun (11/N)

### PR DESCRIPTION
**Pull request by runjin326**
#### What scientific question is your analysis addressing?
Modify the module to re-run for v11 base histology file.

#### What was your approach?
1) Modify directories and names for data input 
2) Re-run with v11 base file

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/303

#### Which areas should receive a particularly close look?
1) Note that TARGET samples are included in the output files.
2) Note that 3 samples `BS_07NXQC7P`, `BS_GGY4Q2C9` and `BS_SV4V3N4Y` were not present in the output file any more.

#### Is there anything that you want to discuss further?
Note that for one sample, the following changes are seen:
From: 
```
PT_X648RVMK 7316-238 BS_GGY4Q2C9 BS_N6N147BY CNS Embryonal, NOS
```
To:
```
PT_X648RVMK 7316-238 NA BS_N6N147BY ETMR, C19MC-altered
```
Please check to see whether this is expected.

Also note that LIN28A z-score are pretty different compare to previous results. 

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes.

#### What types of results are included (e.g., table, figure)?
```
results/biospecimen_ids_embryonal_subtyping.tsv
results/cleaned_chr19_cn.tsv
results/embryonal_tumor_molecular_subtypes.tsv
results/embryonal_tumor_subtyping_relevant_data.tsv
```

#### What is your summary of the results?
NA

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-embryonal/03-clean-c19mc-data.Rmd [hunk 1]
```diff
@@ -100,15 +100,15 @@ data_dir <- file.path("..", "..", "data")
...
 # full clinical file
-histologies_file <- file.path(data_dir, "histologies-base-adapt.tsv")
+histologies_file <- file.path(data_dir, "histologies-base.tsv")
 histologies_df <- read_tsv(histologies_file,
                            col_types = cols(
                              molecular_subtype = col_character()
                            ))
 
 # copy number consensus file (analysis file)
 # TODO: get this from data download
```
Comment (jharenza) [inline analyses/molecular-subtyping-embryonal/03-clean-c19mc-data.Rmd]:
```suggestion
```

**Review by jharenza — CHANGES_REQUESTED** [review]
This is going to need some updates. I see `TARGET-20-PADYIR-04A-01R`, an AML, being subtyped here, but should not be included. Same with `TARGET-30*`, so we should modify the samples to not have these samples in this specific CNS subtyping module.

EDIT: how about we remove TARGET cohort as exclude?

**Comment by runjin326** [comment]
@jharenza - I took a look at the code and the reason for having the AML (TARGET 20*) is because that sample has fusion event. And based on this line:
```
all_biospecimen_ids <- 
  unique(c(filtered_embryonal_df %>% pull(Kids_First_Biospecimen_ID),
           fusion_biospecimens))
```
the sample is added.

So even if we set up cohort requirement, it will still be added back. So I added additional filters:
```
histologies_df %>%
  dplyr::filter(Kids_First_Biospecimen_ID %in% all_biospecimen_ids) %>%
  dplyr::filter(!cohort %in% c("GMKF", "TARGET")) %>% 
  dplyr::select(Kids_First_Biospecimen_ID) %>%
  write_tsv(output_file)
```

This removes other GMKF/TARGET samples. However, note that the 3 samples that I commented before are still missing - and that causes deletions of these two lines + change described above.
```
PT_9BZETM0M	7316-158	BS_SV4V3N4Y	BS_5P1TN10Z	CNS Embryonal, NOS
PT_9BZETM0M	7316-158	BS_SV4V3N4Y	BS_TV5B86ZD	CNS Embryonal, NOS
```

**Comment by jharenza** [comment]
>Note that 3 samples BS_07NXQC7P, BS_GGY4Q2C9 and BS_SV4V3N4Y were not present in the output file any more.

This is OK - it looks like maybe these were old samples we removed from the releases in general, as we do not have them in OpenPBTA v22, either.

**Review by jharenza — APPROVED** [review]
z-scores don't look terrribly different than before, but also can be due to the 2 RNA samples missing now.

**Pull request merged**